clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -665,7 +696,7 @@ void setInFlightTransactionalRequestCorrelationId(int correlationId) {
         inFlightRequestCorrelationId = correlationId;
     }
 
-    void clearInFlightTransactionalRequestCorrelationId() {
+    private void clearInFlightTransactionalRequestCorrelationId() {
```

Comment:
Probably clear enough if it's just `clearInFlightCorrelationId`? Maybe we can rename the field similarly if it seeems reasonable.

Reply:
Yep, sounds good.